cmd/testutil/json2yaml/json2yaml.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
Update the PR title to match the first PR.

Reply:
Done, thanks!

Reply:
Done, thanks!